### PR TITLE
add prop disableSwipe

### PR DESCRIPTION
**Pull request by Duya3fithou**

### CODE REVIEW

**Comment by esthor** [comment]
Thank you for the PR!

This looks simple enough. Just curious, why would you want to disable swiping? Do you have a use case?

File: Swipeable/SwipeableFlatList.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ import SwipeableRow from './SwipeableRow';
 //      * Callback method to render the view that will be unveiled on swipe
 //      */
 //     renderQuickActions: renderItemType,
+//     disableSwipe?: ?boolean,
```
Comment (esthor) [inline Swipeable/SwipeableFlatList.js:30]: Could you also add a line in the README.md about this new prop so it's not undocumented. 🙏

**Comment by Duya3fithou** [comment]
> Thank you for the PR!
> 
> This looks simple enough. Just curious, why would you want to disable swiping? Do you have a use case?

Yeah, I use your library to render a list, where the user can perform actions such as editing or deleting. 
However, at another entry point, or in another user role, only views can be performed, but other actions cannot be performed. 
Reusing the old screen with the disableSwipe prop is completely reasonable and saves resources.

File: Swipeable/SwipeableFlatList.js (hunk 1 above)
Comment (Duya3fithou) [reply Swipeable/SwipeableFlatList.js:30]: DONE

**Comment by esthor** [comment]
I think I largely understand that use case.

In those cases, it sounds like you are basically looking for a `<FlatList/>`. Would that work for your use case?


Basically, in your component render/return, something like:
```
...
return(

if (userHasActionsPermissions) {
<SwipeableList
renderQuickActions=...
maxSwipeDistance=...
boundFirstRowOnMount=...
...restOfListProps
/>
}

if (!userHasActionsPermisssions) {
<FlatList
...restOfListProps
/>
}

)
```

**Pull request closed without merging**